### PR TITLE
Unattended improvements 4 3

### LINKED ISSUE #1561: Failed to reinstall wazuh dashboard in verbose mode
|Wazuh version|Install type|Action performed|Platform|
|---|---|---|---|
| 4.3 | Assistant | Install | General |

When  performing a reinstallation of a specific component using the verbose mode, it has been indicated to me that they are not the correct parameters, this should be reviewed since we should allow this type of execution or at least, clarify the order of these specific parameters

image

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
|Closes #1561 #1560 #1559 #1558 |

## Description

This PR fixes multiple issues in the Wazuh installation assistant.

## Tests

https://devel.ci.wazuh.info/job/Test_unattended_distributed/467/
https://ci.wazuh.info/job/Test_unattended/83/
https://ci.wazuh.info/job/Test_unattended/84/

### CODE REVIEW

File: unattended_installer/install_functions/checks.sh [hunk 1]
```diff
@@ -193,7 +193,7 @@ function check_dist() {
         notsupported=1
     fi
     if [ -n "${notsupported}" ] && [ -z "${ignore}" ]; then
-        common_logger -e "The system is not supported, supported systems are: Red Hat Enterprise Linux 7, 8; CentOS 7, 8; Amazon Linux 2; Ubuntu 16.04, 18.04, 20.04"
+        common_logger -e "The system is not recommended, recommended systems are: Red Hat Enterprise Linux 7, 8; CentOS 7, 8; Amazon Linux 2; Ubuntu 16.04, 18.04, 20.04. Use -i|--ignore-check to skip this comprobation."
```
Comment (rauldpm) [inline unattended_installer/install_functions/checks.sh:196]: Maybe we should avoid to say "is not recommended", we should avoid negative connotations

**Review by rauldpm — CHANGES_REQUESTED** [review]
Please review the message

File: unattended_installer/install_functions/checks.sh (hunk 1 above)
Comment (alberpilot) [reply unattended_installer/install_functions/checks.sh:196]:
```suggestion
        common_logger -e "The recommended systems are: Red Hat Enterprise Linux 7, 8; CentOS 7, 8; Amazon Linux 2; Ubuntu 16.04, 18.04, 20.04. The current system doesn't match this list. Use -i|--ignore-check to skip this comprobation."
```

**Review by rauldpm — APPROVED** [review]
LGTM, great job!

**Review by alberpilot — APPROVED** [review]

**Pull request merged**